src/main.cpp

```
@@ -55,13 +51,30 @@ int main(int argc, char *argv[])
+    {
+        bool cliMode = false;
+        QString execName = QFileInfo(QString(argv[0])).fileName().toLower();
+        if (execName.startsWith("qtmesh") && !execName.contains("editor")) {
+            cliMode = true;
+        }
+        for (int i = 1; i < argc; ++i) {
+            QString arg(argv[i]);
+            if (arg == "--cli") { cliMode = true; break; }
+        }
+        if (!cliMode && argc > 1) {
+            QString cmd(argv[1]);
```

Cppcheck (2.19.0)

[error] There is an unknown macro here somewhere. Configuration is required. If slots is a macro then please configure it.

(unknownMacro)